### DISCUSSION TITLE
[BUG] Getting invalid json errors with local qwen 2.5 33b after v0.4.0 update

### DISCUSSION THREAD

**Discussion by murtaza-nasir** [open-webui/open-webui#7065] [Issues]
## Installation Method

I am running Open WebUI using the `docker-compose.yml` file provided in the repository.

## Environment

- **Open WebUI Version:** v0.4.0
- **Ollama (if applicable):**  v0.2.0

- **Operating System:** Ubuntu 24.04.1 LTS (GNU/Linux 6.8.0-48-generic x86_64)
- **Browser (if applicable):** Firefox 119.0

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I expect to be able to connect to and use my locally running instance of qwen 2.5 33b coder through Ollama like I was able to yesterday before updating.

## Actual Behavior:

I receive the following error message: 

"Uh-oh! There was an issue connecting to Local-large-programmer.
Unexpected token '<', "<html>
<h"... is not valid JSON"

## Description

**Bug Summary:**
Cannot connect to qwen 2.5 33b coder running locally with Ollama after updating to Open WebUI v0.4.0. This was working yesterday on the previous version.

## Reproduction Details

**Steps to Reproduce:**

1.  Update Open WebUI to v0.4.0 using docker-compose pull and docker-compose up -d.
2.  Ensure qwen 2.5 33b coder is running in Ollama.
3.  Attempt to connect to "Local-large-programmer" in Open WebUI.
4.  Observe the error message.

## Logs and Screenshots

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/e93368c2-bd25-4eee-8f91-65e0b3cacef6)

## Additional Information

- This issue only occurs with qwen 2.5 33b coder. Claude sonnet 3.5 has been producing code without any problems.
- The qwen 2.5 33b coder model was working correctly with the previous version of Open WebUI.
- I have confirmed that the qwen 2.5 33b coder model is running correctly in vllm.

## Note

I have tried restarting both Open WebUI and vllm, but the issue persists.

**Comment by tjbck** [comment]
![image](https://github.com/user-attachments/assets/27801991-684b-4794-9d54-d05832dd0b05)

Unable to reproduce the issue, keep us updated!

**Comment by murtaza-nasir** [comment]
This most likely has something to do with the citation capability that was enabled by default after update. I turned it off and it looks like it is fine now. I'm surprised though since qwen2.5 coder is very good at tasks like this. This only happened with very long prompts (I only tried either prompts with ~50K tokens or very short prompts). The short prompts didn't seem to have this issue. Its also generating something thats causing 404 errors from the vllm api.
